common-lib/src/cmr/common/util.clj

```diff
@@ -469,6 +469,17 @@
     (catch NumberFormatException _
       false)))
 
+;; Differences from numeric-string? above:
+;;   More strict - For example, returns false on string "0D".
+;;   Works on any sequence of characters, not just strings.
+(defn numeric? [s]
+  (if-let [s (seq s)]
```

Comment:
This should be a `when-let` since there is no `else` branch

Reply:
good catch, done